### ISSUE TITLE
🛑 Daily Loss Limits & Trading Controls

### ISSUE THREAD

**Issue by abhidp** [abhidp/quant-trading-engine#33]
**Priority:** HIGH  
**Estimate:** 1-2 hours  

## Problem
No daily limits - can lose unlimited amount per day.

## Solution
Stop trading when daily loss exceeds 3% of account balance.

## Acceptance Criteria
- [ ] Track daily P&L from start of trading session
- [ ] Stop all trading when daily loss > 3% of account balance
- [ ] Limit maximum 5 open positions at any time
- [ ] Prevent new trades if max positions reached
- [ ] Log when limits are hit
- [ ] Reset daily counters at 00:00 server time

## Code Requirements
```python
def check_daily_limits():
    daily_pnl = calculate_daily_pnl()
    account_balance = mt5.account_info().balance
    max_loss = account_balance * -0.03
    
    if daily_pnl < max_loss:
        return False  # Stop trading
    
    open_positions = len(mt5.positions_get())
    if open_positions >= 5:
        return False  # Max positions reached
        
    return True  # OK to trade

def calculate_daily_pnl():
    # Calculate total P&L since 00:00 server time
```
